### ISSUE TITLE
Identify replication factors for Storage across cloud providers

### ISSUE THREAD

**Issue by ericksod** [cloud-carbon-footprint/cloud-carbon-footprint#250]
#### Overview
In our estimations for Storage, we have an opportunity to make it a bit more accurate. For storage when we have a certain amount of usage, there is a difference in that requested usage and the actual infrastructure being allocated (more is allocated for stability, reliability, backup, etc.)- this difference is the replication factor. We would like to take this into account in our calculations.
#### Out of Scope:
Implementing the replication factors
## Acceptance Criteria
- [x] Identify replication factors for AWS services (and add to ACs)
- [x] Identify replication factors for GCP services (and add to ACs)
- [x] Identify replication factors for Azure services (and add to ACs)
- [ ] Update [implementation ticket](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/232) to include all factors we need to update
- [ ] Document replication factors (and assumptions?) in a public spreadsheet? or existing?

## Tech Tasks
- [ ] Run Changesets, bump any packages that have been updated/changed

## Review Tasks
- [ ] Update methodology to indicate the use of replication factors for storage. Link to public spreadsheet?

**Comment by ccasher** [comment]
Original Issue: https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/issues/211

AWS Storage Services:
https://docs.aws.amazon.com/whitepapers/latest/aws-storage-services-overview/introduction.html

**Comment by aalva05-tw** [comment]
Here is the spreadsheet with the values: 
https://docs.google.com/spreadsheets/d/1JvSM2Z61hurpDl5uJvqrc5QoTyWYP6yLrQZ8tId3KOU/edit?usp=sharing

**Comment by dtoakley** [comment]
I have reviewed the spreadsheet, and added additional columns E/F for general notes and implementation notes as it relates to the Cloud Provers API responses for billing data, as well as a number of missing services. 

One thing to note is that replication doesn't only exist with Storage Services, but also sometimes with Database Services. For example here (https://docs.aws.amazon.com/index.html), we had defined the replication factors for Storage services, however we were missing some of the Database services, which can also have default replication factors. I've added most/all of these for AWS and GCP. Azure still needs a bit of research, but I'm not sure that needs to holds us back in implementing AWS/GCP, at least to begin with to get the right code structure/patterns in place.

Another notes i that replication isn't just for usage types we classify as storage. For example if an RDS instance is replicated as part of a Multi-AZ deployment, then it will have both storage and compute replication as this service consumes both usage types. I didn't have time to verify this for all services -- it'll probably require some query of the provider billing APIs to check and confirm, but it's likely just for Database Services as they often have both storage and usage rows returned from the API. Hope that makes sense :)
